doc/assessment.md

```diff
@@ -117,12 +244,18 @@ Course staff see any reported issues show up on the "Issues" tab.
 
 ![Issue report](assessment-report4.png)
 
-## Access control
+## `constantQuestionValue`: Points boost for correct Homework answers
 
-See the [Access control page](accessControl.md) for details.
+FIXME
 
-By default, an assessment is only accessible to `Instructor` users. To change this, the `allowAccess` option can be used in the assessment's `infoAssessment.json` file.
+## Changing assessments while students are working on them
 
-## Adding text and links to assessments
+While it is simplest to not change an assessment while students are working on it, we know that there are sometimes good reason that changes are needed. For example, a question might need to be removed because it contains an error, new questions might be added, or the points might be changed.
 
-See the [`clientFiles` and `serverFiles`](clientServerFiles.md) page for details, and [`exam1` in the example course](https://github.com/PrairieLearn/PrairieLearn/blob/master/exampleCourse/courseInstances/Sp15/assessments/exam1/) for an example.
+PrairieLearn’s general philosophy is to try and do the right thing in these situations. It will never automatically reduce a student’s total assessment “points” or “percentage score” due to assessment changes. Points that have been given to a student will not be taken away (unless the instructor explicitly edits the student points).
+
+The precise rules for updating assessment instances differ between `Homework` and `Exam` assessments.
+
+**`Exam` assessment updates:** Exam assessment instances are generated when the student starts the exam, and they are never automatically deleted, regenerated, or updated, even when the original assessment is changed in some way. This is a safety mechanism to avoid having students' assessments changed during an exam. However, if you want to force the regeneration of assessment instances then you can do so with the “reset” button on the instructor view of the assessment. While writing and testing an assessment you might need to do this many times. Once an assessment is live, you should of course be very careful about doing this (basically, don’t do it on a production server once a student is working on an assessment).
+
+**`Homework` assessment updates:** New questions added to Homeworks will be automatically integrated into student homeworks that are currently in progress, and removed questions will be removed from students. Updates to `maxPoints` will take effect the next time a student grades a question.
```

Comment:
Consider adding a disclaimer to speak with PrairieLearn developers and/or support staff when doing this?

Reply:
I suspect everyone is just going to do it anyway 😄

I’ll see if I can add some more scary language warning people to not make major changes to live exams.